hw/hw5/Hw5_t1_oop1.py

```diff
@@ -0,0 +1,67 @@
+"""Homework 5.1."""
+import datetime
+import sys
+from typing import Any, Optional
+
+
+class Student:
+    """Create student."""
+
+    def __init__(self, first_name: str, last_name: str) -> None:
+        self.first_name = first_name
+        self.last_name = last_name
+
+    def do_homework(self, homework: "Homework") -> Optional[Any]:
+        """Сheck if homework is done.
+
+        Args:
+            homework: Instance of class Homework
+
+        Returns:
+            The return value. If the time for completing the task has not expired - return instance of class Homework
+                            else - print the message to sys.stdout: "You are late" and return None
+        """
+        self.homework = homework
+        if self.homework.is_active():
+            return self.homework
+        sys.stdout.write("You are late")
+        return None
```

Comment:
Here we can leave function without return None statement, because function returns it by default